packages/build/src/download-center.ts

```diff
@@ -114,6 +117,40 @@ const uploadToDownloadCenter = (s3: S3, config: string): Promise<any> => {
   return upload(uploadParams, s3);
 };
 
+const delay = (msecs): any => new Promise((resolve) => {
```

Comment:
This could be 

```suggestion
const delay = (msecs): Promise<void> => new Promise((resolve) => {
```

, right?

Reply:
yep!